tests/BulkTest.php

```diff
@@ -765,4 +769,32 @@ public function testHasIndex(): void
         $bulk->setIndex('unittest');
         $this->assertTrue($bulk->hasIndex());
     }
+
+    /**
+     * @group unit
+     */
+    public function testSendRequestEntityTooLargeExceptionIf413Response(): void
+    {
+        $response = new ElasticaResponse('', 413);
+
+        /** @var Client|MockObject $clientMock */
+        $clientMock = $this->createMock(Client::class);
+        $clientMock
+            ->method('request')
+            ->willReturn($response);
+
+        $index = $this->_createIndex();
```

Comment:
this is marked as an @unit test, so the ES server is not running.
is the 'index' really needed?